### ISSUE TITLE
Could not find function "%^%"

### ISSUE THREAD

**Issue by yaxing0zhao** [tomis9/markovClust#2]
Hi tomis9, I want to use your markovClust package, but there ia an error: could not find function "%^%". Could you please reply me what the meaning of "%^%" is and where it comes from? Does it need more package when I run it? Appreciate it in advance.

**Comment by tomis9** [comment]
Hello yaxing0zhao, 
Function %^% comes from package "expm" and, according to it's documentation, it "Computes the k-th power of a matrix." Installing this package with `install.packages(expm)` and loading it with `library(expm)` should solve the problem.
I was hoping that including "#' @import expm" in markovClust's function description would force R to install this dependency automatically. As I checked it recently, it does not work. 

Most probably I will not develop this package anymore nor debug it, but I will be happy to answer any further questions.
Cheers

PS. The best reference for Markov clustering algorithm I found is "Data Mining and Analysis, Fundamental Concepts and Algorithms" by Zaki, Meira. This package is based on this book.
